### PR TITLE
SHOUT OUT TO THE CUSTOMIZER

### PR DESCRIPTION
**Pull request by sampoder**
also @jasonappah I've said this before but your app is very cool, hope this credits you well

### CODE REVIEW

**Comment by lachlanjc** [comment]
This is awesome. Can you link to Jason’s scrapbook instead of GitHub?

**Comment by sampoder** [comment]
good thinking

**Comment by lachlanjc** [comment]
Actually using a Mention component would be even more fab

**Comment by sampoder** [comment]
a bit like that?

**Comment by sampoder** [comment]
oops I forgot to import

File: pages/about.mdx [hunk 1]
```diff
@@ -35,6 +36,8 @@ Behind the scenes, the site ([all open source!](https://github.com/hackclub/summ
 
 To customize the CSS on your profile page, run `/scrappy setcss <link>` in Slack, giving a link to a CSS file or a [GitHub Gist](https://gist.github.com). [Here’s the default CSS](https://scrapbook.hackclub.com/themes/default.css), for your overwriting pleasure.
 
+Want to preview your CSS before adding it to your profile? Check out <Mention username={jasonappah} />'s Scrapbook Customizer [here](https://scrapbook.hackclub.com/customizer)
```
Comment (lachlanjc) [inline pages/about.mdx:39]: This is referencing his name as a variable—you’ll need a string instead
```suggestion
Want to preview your CSS before adding it to your profile? Check out <Mention username="jasonappah" />’s [Scrapbook Customizer](https://scrapbook.hackclub.com/customizer).
```

**Comment by lachlanjc** [comment]
Thanks!

**Comment by jasonappah** [comment]
Thanks everyone!

**Pull request merged**